src/components/linePercentage/LinePercentage.tsx

```diff
@@ -0,0 +1,71 @@
+import { classNameGenerator } from 'utils/pureFunctions';
+
+export interface LinePercentageData {
+  color: string;
+  decPercent: number;
+  label?: string;
+}
+
+interface LinePercentageProps {
+  percentages?: LinePercentageData[];
+  showEmpty?: boolean;
+}
+
+export const LinePercentage = ({
+  percentages,
+  showEmpty,
+}: LinePercentageProps) => {
+  const linePercentages = (percentages: LinePercentageData[]) => {
```

Comment:
I think there's no reason for this function (`linePercentages`) to be within the component - it makes it get re-declared every render. This is a pure function that can be outside the component, e.g. line 13

Reply:
Placed outside, not in pureFunctions since it is very specific for this component

Reply:
@RanCohenn I didn't mean Pure Component or any such thing. Just a pure function - it doesn't have side effects, it does the exact same things for a given input.